src/fetching/tasks/getAllTasks.ts

```diff
@@ -0,0 +1,10 @@
+import { defaultFetch } from '..';
+import { ZetkinTask } from '../../types/zetkin';
+
+export default function getAllTasks(orgId : string, fetch = defaultFetch) {
```

Comment:
I think it's clearer to call this `getOrganizationTasks` but that's a small thing, up to you if you want to change it.

Reply:
I will change it, and add to another issue the larger problem of consistency when naming these fetch functions, and also naming the reactQuery keys... #253